google-cloud-clients/google-cloud-bigtable/README.md

```diff
@@ -130,6 +130,66 @@ try {
 }
 ```
 
+## Opencensus Tracing
+
+Cloud Bigtable client supports [Opencensus Tracing](https://opencensus.io/tracing/),
+which gives insight into the client internals and aids in debugging production issues.
+By default, the functionality is disabled. To enable, you need to add a couple of
+dependencies and configure an exporter. For example to enable tracing using 
+[Google Stackdriver](https://cloud.google.com/trace/docs/):
+
+[//]: # (TODO: figure out how to keep opencensus version in sync with pom.xml)
```

Comment:
I'm not sure how to ensure that the opencensus versions in this readme stay up to date with google-cloud-clients/pom.xml properties. I thought about using version markers, but those seem to only be used for internal artifacts. It would be awesome if version.txt could be generalized to manage static external versions as well.